src/pages/Login/Login.js

```diff
@@ -1,7 +1,102 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import './Login.scss';
 
 const Login = () => {
-  return <div>login</div>;
+  const navigate = useNavigate();
+  // 로그인 input 값 받기
+  let [inputs, setInputs] = useState({
```

Comment:
```suggestion
  const [inputs, setInputs] = useState({
```
react 의 동작원리를 잘 생각해보세요.
네이밍도 깊게 고민해보세요!